homeassistant/components/horizon/media_player.py

```diff
@@ -63,17 +64,23 @@ class HorizonDevice(MediaPlayerDevice):
     """Representation of a Horizon HD Recorder."""
 
     def __init__(self, client, name, keys):
-        """Initialize the remote."""
+        """Initialize the entity."""
         self._client = client
         self._name = name
+        self._icon = DEFAULT_ICON
         self._state = None
         self._keys = keys
 
     @property
     def name(self):
-        """Return the name of the remote."""
+        """Return the name of the entity."""
         return self._name
 
+    @property
+    def icon(self):
+        """Return the icon of the entity."""
+        return self._icon
```

Comment:
We can return the constant here directly.

Reply:
oh of course, fixed now